qubesmanager/backup.py

```diff
@@ -21,379 +20,299 @@
 #
 #
 
-import sys
-import os
-import signal
-import shutil
-from PyQt4.QtCore import *
-from PyQt4.QtGui import *
+import traceback
 
-from qubes.qubes import QubesVmCollection
-from qubes.qubes import QubesException
-from qubes.qubes import QubesDaemonPidfile
-from qubes.qubes import QubesHost
-from qubes import backup
-from qubes import qubesutils
+import signal
 
-import qubesmanager.resources_rc
+from qubesadmin import Qubes, exc
+from qubesadmin import utils as admin_utils
+from qubes.storage.file import get_disk_usage
 
-from pyinotify import WatchManager, Notifier, ThreadedNotifier, EventsCodes, ProcessEvent
+from PyQt4 import QtCore  # pylint: disable=import-error
+from PyQt4 import QtGui  # pylint: disable=import-error
+from . import ui_backupdlg  # pylint: disable=no-name-in-module
+from . import multiselectwidget
 
+from . import backup_utils
+from . import utils
+import grp
+import pwd
+import sys
+import os
+from . import thread_monitor
+import threading
 import time
-from .thread_monitor import *
-from operator import itemgetter
-
-from datetime import datetime
-from string import replace
 
-from .ui_backupdlg import *
-from .multiselectwidget import *
 
-from .backup_utils import *
-import main
-import grp,pwd
+class BackupVMsWindow(ui_backupdlg.Ui_Backup, multiselectwidget.QtGui.QWizard):
 
-
-class BackupVMsWindow(Ui_Backup, QWizard):
-
-    __pyqtSignals__ = ("backup_progress(int)",)
-
-    def __init__(self, app, qvm_collection, blk_manager, shutdown_vm_func, parent=None):
+    def __init__(self, app, qvm_collection, parent=None):
         super(BackupVMsWindow, self).__init__(parent)
 
         self.app = app
         self.qvm_collection = qvm_collection
```

Comment:
This may be misleading, as in many places `app` is used for `Qubes()`. IMO better use `qt_app` and `qubes_app` (or just `app`).